src/PowerShellEditorServices.Protocol/Server/LanguageServer.cs

```diff
@@ -1755,13 +1755,15 @@ private static FileChange GetFileChangeDetails(Range changeRange, string insertS
                 {
                     // Semantic markers aren't available if the AnalysisService
                     // isn't available
-                    semanticMarkers = new ScriptFileMarker[0];
+                    semanticMarkers = new List<ScriptFileMarker>();
                 }
 
+                scriptFile.SyntaxMarkers.AddRange(semanticMarkers);
```

Comment:
Do we really want to modify the SyntaxMarkers collection in ScriptFile by adding semantic markers?  Maybe we should have two collections in ScriptFile (SyntaxMarkers and SemanticMarkers) or perhaps rename the property to DiagnosticMarkers (or something like that).

Reply:
I went with the rename since the syntax markers and semantic markers are both treated the same when they get sent to vscode.